policy/approval/approve.go

```diff
@@ -103,9 +103,17 @@ func (r *Rule) Evaluate(ctx context.Context, prctx pull.Context) (res common.Res
 	res.Name = r.Name
 	res.Description = r.Description
 	res.Status = common.StatusSkipped
+	res.Requires = common.Actors{
+		Organizations: r.Requires.Organizations,
+		Teams:         r.Requires.Teams,
+		Users:         r.Requires.Users,
+	}
+
+	var predicatesInfo []*common.PredicateInfo
 
 	for _, p := range r.Predicates.Predicates() {
-		satisfied, desc, err := p.Evaluate(ctx, prctx)
+		satisfied, pPredicateInfo, err := p.Evaluate(ctx, prctx)
+		predicatesInfo = append(predicatesInfo, pPredicateInfo)
```

Comment:
Since now we are putting the `satisfied` as a bool field to `PredicateResult`, the pointer would not be `nil` anymore given our logic. So I assume we don't need to check this anymore?

Reply:
Yeah, that's correct